HISTORY.md

```diff
@@ -1,5 +1,183 @@
 # Release Notes
 
+## 1.0.0 - 2023-03-28
+
+This is a major release that introduces a new API to the `SDV` aimed at streamlining the process of synthetic data generation! To achieve this, this release includes the addition of several large features.
+
```

Comment:
I think some sub-headers may actually be really nice here.

Eg. This section can be 
```
### Metadata
```